crates/runtime/src/podswatcher.rs

```diff
@@ -56,20 +51,25 @@ impl PodsWatcher {
                         }
 
                         if let Ok(app) = App::new(root_path.clone()) {
-                            event_handler(PodsWatcherEvent::PodsUpdated(app));
+                            futures::executor::block_on(async {
```

Comment:
We should execute this in the tokio runtime - we should already be in that context

Reply:
This is actually context of sync watcher callback. Can we keep this for tomorrow/demo and I'll update it to use mpsc channel and tokio runtime?

Reply:
I actually found there is tx.blocking_send so we don't need a futures::executor. Updated